pkgs/by-name/li/libfaketime/package.nix

```diff
@@ -9,18 +9,26 @@
 
 stdenv.mkDerivation rec {
   pname = "libfaketime";
-  version = "0.9.10";
+  # 0.9.10 break dict-db-wiktionary and quartus-prime-lite on linux,
+  # and 0.9.11 break everything on darwin
+  version = if stdenv.hostPlatform.isDarwin then "0.9.10" else "0.9.11";
 
   src = fetchFromGitHub {
     owner = "wolfcw";
     repo = "libfaketime";
     rev = "v${version}";
-    sha256 = "sha256-DYRuQmIhQu0CNEboBAtHOr/NnWxoXecuPMSR/UQ/VIQ=";
+    sha256 =
+      if stdenv.hostPlatform.isDarwin then
```

Comment:
Nit (not blocker): I would have written this (and the next) conditional on the version -- using hostPlatform feels indirect / more difficult to read to me.